src/pages/management/configuration-api/v3.5/index.mdx

```diff
@@ -2909,6 +2909,51 @@ curl -X GET \
 
 </Section>
 
+<Section>
+
+<Text>
+
+### Get Product Limits for Plan
+
+Each pricing plan has different resource limits (number of groups, agents). When the user wants to downgrade, this method allows checking what resources exceeded the limits for a given plan.
+
+#### Specifics
+
+|             |                                                                             |
+| ----------- | --------------------------------------------------------------------------- |
+| HTTP Method | **GET**                                                                     |
+| Method URL  | `https://api.livechatinc.com/v3.5/configuration/action/check_product_limits_for_plan` |
+| Required scopes | `subscription.lc:rw` |
+
+
+#### Request
+
+| Query string | Required | Data type | Notes |
+| - | - | - | - |
+| `plan`| yes | `string`| Possible values: `starter`, `team`, `enterprise`, `enterpriseplus`      |
```

Comment:
I'm pretty confused by the spacing here. Please consult (with Ola I guess) if we should format the tables in plain text (so that each column has a constant width) or not

Reply:
It's not required to have the same width for called ([more details](https://www.markdownguide.org/extended-syntax/#tables)). To keep the text consistent with other parts, I improved spacing [here](https://github.com/livechat/livechat-public-docs/pull/1099/commits/cced25b39699d5f5250b547e992f3cf0d9767df6)